File: pitch/src/login_spec.js
describe('Login', function() {
    beforeEach(function() {
        cy.fixture("basic_user.json").as("user")
        cy.visit("/")
    });

    it('logs me in', function() {
        cy.get('nav').contains('a', 'Login').click();

        cy.get('input[name=email]').type(this.user.email);
        cy.get('input[name=password]').type('12345678');
        cy.contains('input', 'Login').click();

        cy.get('nav').should('contain', this.user.email);
    })
})

File: pitch/src/login_spec.rb
RSpec.describe "Logging in" do
  let!(:user) { FactoryBot.create(:user) }

  it "logs you in" do
    visit root_path
    click_on "Login"

    fill_in "Email", with: user.email
    fill_in "Password", with: "12345678"
    click_button "Login"

    expect(find("nav").text).to include(user.email)
  end
end
